README.md

```diff
@@ -1,4 +1,7 @@
 # OCaml-migrate-parsetree
+[Deprecated] This package is deprecated. Please, use
+[ppxlib](https://github.com/ocaml-ppx/ppxlib) instead.
```

Comment:
You gave some context in the PR, maybe it's worth linking to it?

```suggestion
[ppxlib](https://github.com/ocaml-ppx/ppxlib) instead. See [#126](https://github.com/ocaml-ppx/ocaml-migrate-parsetree/pull/126).
```

Reply:
Yes, good idea! I've just opened a PR for an ocaml.org Changelog entry about the deprecation: https://github.com/ocaml/ocaml.org/pull/1712 . I'll link to the Chagelog entry here once that's merged. That contains more detail than the PR description.